### DISCUSSION TITLE
Clarification/Better way of doing SpamAccountsList.txt

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#316] [Q&A] [answered]
@ThioJoe As you mentioned in the SpamAccountsList.txt other users can have the same username, so does it detect anyone with that name a scammer? Also, a better way of doing SpamAccountsList.txt and SpamThreadsList.txt is to list the scammers channel id instead of username according to me, if this is the case.

**Comment by ThioJoe** [comment, marked as answer]
It's not for YouTube account usernames, it's for spammers who mention stuff like "contact [whatever username] on instagram for hacks" and stuff

**Reply by KendallDoesCoding** [reply]
> It's not for YouTube account usernames, it's for spammers who mention stuff like "contact [whatever username] on instagram for hacks" and stuff

Oh ok makes sense!
